### PR TITLE
BTRX-669-Change Field in Admin Tab

### PR DESCRIPTION
**Pull request by wloforteBroad**
## Addresses

https://broadinstitute.atlassian.net/browse/BTRX-669

Admin Tab: Please add the Drop-down feature under "Type of Initial Review" (current functionality in production under Review Category) Expedited, Exempt, Full Board, Not Human Subjects Research, Not Engaged* (*new choice).

---

- [ ] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Get a thumb from @rushtong
- [ ] **Submitter**: Merge to develop using github's "Squash and Merge" feature
- [ ] **Submitter**: Delete branch after merge

### CODE REVIEW

File: src/main/webapp/adminOnly/AdminOnly.js [hunk 1]
```diff
@@ -184,7 +185,7 @@ class AdminOnly extends Component {
     form.investigatorFirstName = this.state.formData.investigatorFirstName;
     form.investigatorLastName = this.state.formData.investigatorLastName;
     form.initialDate = this.parseDate(this.state.formData.initialDate);
-    form.initialReviewType = this.state.formData.initialReviewType;
+    form.initialReviewType = JSON.stringify(this.state.formData.initialReviewType);
```
Comment (rushtong) [inline src/main/webapp/adminOnly/AdminOnly.js]: Curious about why this needs to be stringified - usually that's for display purposes. I'm pretty sure it doesn't do anything structurally to the data here.

Comment (dags) [reply src/main/webapp/adminOnly/AdminOnly.js]: Select component returns selected value as an json object { label:"xxxx", value: "zzz"}. If we store only selectedValue.value then we need to recreate selected Value at read. We saved as stringified, we parse at read.

Comment (wloforteBroad) [reply src/main/webapp/adminOnly/AdminOnly.js]: @rushtong can we merge this PR?

Comment (rushtong) [reply src/main/webapp/adminOnly/AdminOnly.js]: Absolutely ... I approved it!

**Pull request merged**